File: backend/Services/Parsers/AntiwordleParser.cs
using System.Text.RegularExpressions;

namespace backend.Services.Parsers
{
    public class AntiwordleParser : BasicScoreResultParser
    {
        private ILogger<AntiwordleParser> _logger;

        public AntiwordleParser(ILogger<AntiwordleParser> logger) : base(logger)
        {
            _logger = logger;
        }

        public override bool CountWinner => false;
        public override string GameName => "Antiwordle";
        public override bool GolfScoring => false;
        public override string? HelpText => null;
        protected override Regex Parser => new Regex($"{GameName} #\\d+\\s*(?<{ScoreGroup}>\\d+) guesses", RegexOptions.Singleline);
        protected override string? ExtraContent => null;
        public override string Url => "https://www.antiwordle.com/";
    }
}

File: lambdas/src/parsers/antiwordle-parser.ts
import { BasicScoreResultParser } from './base/basic-score-parser';

export class AntiwordleParser extends BasicScoreResultParser {
  readonly countWinner = true;
  readonly gameName = 'Antiwordle';
  override readonly golfScoring = false;
  readonly helpText = null;
  readonly url = 'https://www.antiwordle.com/';
  protected readonly extraContent = null;
  
  protected readonly parser = new RegExp(
    `${this.gameName} #\\d+\\s*(?<${BasicScoreResultParser.SCORE_GROUP}>\\d+) guesses`,
    's'
  );
}
